crates/sage-cli/src/output.rs

```diff
@@ -161,12 +163,14 @@ impl Runner {
             "peptide",
             "proteins",
             "num_proteins",
+            "start_positions",
+            "end_positions",
             "filename",
-            "scannr",
+            "scan",
             "rank",
             "label",
-            "expmass",
-            "calcmass",
+            "measured_mass",
```

Comment:
What's the rationale for changing column names?

Reply:
Oh, the rationale here, although I'm not wedded to them, are to 1) keep the underscore delimiting in the rest of the columns, 2) spell out calculated and experimental/measured, and 3) including number after scan doesn't feel necessary since every entry has "scan=[scan number]".

Reply:
I'm not opposed to it - but the column names have been stable for ~2.5 years so I don't see a huge need to rename them. They were originally named like this because sage results (used) to be passed directly to mokapot for rescoring.